standard/statements.md

```diff
@@ -917,7 +917,7 @@ The end point of a `switch` statement is reachable if the switch statement is re
 >         case "circle":
 >             return new Circle(2);
 >         …
->         case var o when (o?.Trim().Length ?? 0) == 0:
+>         case var o when (string.IsNullOrWhiteSpace(o)) == true:
```

Comment:
Could this just be `case var o when string.IsNullableOrWhitespace(o)`? (No need for `== true` or the brackets?)